frontend/src/pages/modelServing/ModelServingContext.tsx

```diff
@@ -132,11 +139,12 @@ const ModelServingContextProvider: React.FC = () => {
         servingRuntimeTemplateDisablement,
         dataConnections,
         refreshAllData,
+        project,
```

Comment:
Add the project to the context to better use it, following the same way on the pipelines page.